temp_examples/scenic-streetmap-overlay/scenic_plot.html

```diff
@@ -0,0 +1,398 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Scenic Ratings of London</title>
+    <meta charset="utf-8">
+    <link href='https://fonts.googleapis.com/css?family=Lora:400,700italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="scenic_plot.css">
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.2/leaflet.css" />
+    <script src="http://cdn.leafletjs.com/leaflet-0.7.2/leaflet.js"></script>
+    <script type="text/javascript" src="http://maps.stamen.com/js/tile.stamen.js?v1.3.0"></script>
```

Comment:
I haven't been able to make this work yet, but there should be a way to have the silver style google map as a background in leaflet. Here is a way to create map styles https://mapstyle.withgoogle.com/ The stamen map is missing some neighborhood names and that would be great to have.
![Screen Shot 2019-11-04 at 17 38 16](https://user-images.githubusercontent.com/8143964/68143709-ee674c80-ff29-11e9-8edc-c8bf364c7926.png)

Reply:
This would be really nice, I think you need a google account to get an API key to use this but I can try look into it